FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.JobManagement/AzureStorageQueueClient.cs

```diff
@@ -0,0 +1,675 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Diagnostics;
+using Azure;
+using Azure.Data.Tables;
+using Azure.Identity;
+using Azure.Storage.Queues;
+using Azure.Storage.Queues.Models;
+using EnsureThat;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations;
+using Microsoft.Health.Fhir.Synapse.JobManagement.Exceptions;
+using Microsoft.Health.Fhir.Synapse.JobManagement.Extensions;
+using Microsoft.Health.Fhir.Synapse.JobManagement.Models;
+using Microsoft.Health.Fhir.Synapse.JobManagement.Models.AzureStorage;
+using Microsoft.Health.JobManagement;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Synapse.JobManagement
+{
+    public class AzureStorageQueueClient<TJobInfo> : IQueueClient
+        where TJobInfo : AzureStorageJobInfo, new()
+    {
+        private readonly TableClient _azureJobInfoTableClient;
+
+        private readonly QueueClient _azureJobMessageQueueClient;
+
+        private readonly ILogger<AzureStorageQueueClient<TJobInfo>> _logger;
+
+        private readonly int _maximumSizeOfAnEntityInBytes = 1024 * 1024;
```

Comment:
Can this be const **MaximumSizeOfAnEntityInBytes**, and add comment for this?

Reply:
Sure, will update them.